### DISCUSSION TITLE
Searching in finder on mounted drives

### DISCUSSION THREAD

**Discussion by ChrisNow75** [iterate-ch/cyberduck#14341] [Q&A] [answered]
Hi guys 
I am struggling to search anything on my mounted drives . no matter if they are NAS or one drive , google drive shares
Seems like finder is not indexing and searching external (remote drives). Any ideas please?

**Comment by dkocher** [comment, marked as answer]
As [documented](https://docs.mountainduck.io/mountainduck/issues/#operating-system-specific-issues) Spotlight or Windows Search do not work on the volumes mounted with Mountain Duck. Spotlight on macOS could eventually be [enabled manually](https://github.com/iterate-ch/docs/issues/314) but you would also need to change the default mount point to omit the `.noindex` suffix we have added in [version](https://mountainduck.io/changelog/) 4.13.5 and later because of issues with CPU usages when Spotlight accidentally indexed large remote volumes.

**Comment by ChrisNow75** [comment]
Hi
Thanks for the info. Is there somewhere  the instruction how to manually enable it and change the mounting point?
Chris

**Comment by Cassie205** [comment]
Information on how to change the mount location can be found in our [documentation](https://docs.cyberduck.io/mountainduck/preferences/#mount-location):


> Volumes are mounted in the `Volumes` folder in the [application support directory](https://docs.cyberduck.io/mountainduck/support/). You can change the default to another folder that is writable.
 
> Note: You cannot set it to /Volumes on macOS Mojave (10.14 or later) where the directory is not writable. The setting is not available in the version from the Mac App Store.
 
> The default is set to `~/Library/Group Containers/G69SCX94XU.duck/Library/Application Support/duck/Volumes.noindex`.

**Comment by ChrisNow75** [comment]
Thank you Sofie
Apreciated your answer. I will dive into documentation and check if works for me
Regards 
Krzysztof Nowakowski

**Reply by dzg** [reply]
Did you succeed in finding a way to be able to search a mounted drive? I'm looking to index Google Drive.
